tests/commands/test_cmd_config.py

```diff
@@ -0,0 +1,73 @@
+# third-party imports
+from click.testing import CliRunner
+
+# local imports
+from shpkpr.cli import cli
+
+
+def test_no_args():
+    runner = CliRunner()
+    result = runner.invoke(cli, ['config'])
+
+    assert result.exit_code == 0
+    assert 'Usage:' in result.output
```

Comment:
These seem quite similar to the integration test PR that's up in #43, how much of an overlap is there between those tests and these tests?